public/types.ts

```diff
@@ -32,6 +32,9 @@ export interface AuthInfo {
 }
 
 export interface ClientConfigType {
+  readonly_mode: {
+    roles: string[];
```

Comment:
you may also want to export this config in `server/index.ts`

Reply:
It's already there.
https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/blob/96ba75a71e75490dc41f89e9d3f8453893eb7a77/server/index.ts#L24

Reply:
but you need to make it available by adding it to exportToBrowser https://github.com/opendistro-for-elasticsearch/security-kibana-plugin/blob/96ba75a71e75490dc41f89e9d3f8453893eb7a77/server/index.ts#L189

Reply:
Got it, fixed.